design/cstor/v1/cstorpool.md

```diff
@@ -247,9 +247,28 @@ type CStorPoolInstanceStatus struct {
 
 // CStorPoolInstanceCapacity stores the pool capacity related attributes.
 type CStorPoolInstanceCapacity struct {
+	// Amount of physical data (and its metadata) written to pool
+	// after applying compression, raid properties, etc..,
+	Used resource.Quantity `json:"used"`
+	// Amount of usable space in the pool after excluding
+	// metadata and raid properties
+	Free resource.Quantity `json:"free"`
+	// Sum of usable capacity in all the data raidgroups
 	Total resource.Quantity `json:"total"`
-	Free  resource.Quantity `json:"free"`
-	Used  resource.Quantity `json:"used"`
+	// ZFSCapacityAttributes contains advanced information about pool capacity details
+	ZFS ZFSCapacityAttributes `json:"zfs"`
+}
+
+// ZFSCapacityAttributes stores the advanced information about pool capacity related
+// attributes
+type ZFSCapacityAttributes struct {
+	// LogicalUsed is the amount of space that is "logically" consumed
+	// by this pool and all its descendents. The logical space ignores
+	// the effect of the compression and copies properties, giving a
+	// quantity closer to the amount of data that applications see.
+	// However, it does include space consumed by metadata.
+	// NOTE: Used and LogicalUsed can vary depends on the pool properties
```

Comment:
n_p: 
`// NOTE: Used and LogicalUsed can vary depends on the pool properties`
------>
`// NOTE: Used and LogicalUsed can vary depending on the pool properties`

Reply:
Also what pool properties? Is it only the Raid configuration or something else... It would be better to put that explicitly.

Reply:
So one example comment that comes to my mind is : 
`// NOTE: Used and LogicalUsed can vary depending on the pool properties e.g RAID configurations etc` 
Mentioning a few examples of those properties can be helpful.

Reply:
Infact : Let us get rid of this note as this is obvious that `Used` and `LogicalUsed` will vary -- hence two diff fields.

Reply:
Removed the note